### PR TITLE
[MM-28781] migrate string refs in channelView

### PR DESCRIPTION
**Pull request by hardikmodi1**
Summary
Migrate string refs in channelView

#### Ticket Link
JIRA - https://mattermost.atlassian.net/browse/MM-28781
ISSUE - Fixes https://github.com/mattermost/mattermost-server/issues/15517

### CODE REVIEW

File: components/channel_view/channel_view.jsx [hunk 1]
```diff
@@ -90,10 +90,12 @@ export default class ChannelView extends React.PureComponent {
             channelId: props.channelId,
             deferredPostView: ChannelView.createDeferredPostView(),
         };
+
+        this.channelView = React.createRef();
```
Comment (Willyfrog) [inline components/channel_view/channel_view.jsx:94]: Could you rename it to 
```suggestion
        this.channelViewRef = React.createRef();
```

**Review by Willyfrog — CHANGES_REQUESTED** [review]
thanks for contributing! Looking good, just a minor nit for consistency with other PRs

**Review by Willyfrog — COMMENTED** [review]
lgtm! thanks for contributing

**Review by Willyfrog — APPROVED** [review]

**Review by sudheerDev — APPROVED** [review]
LGTM

**Review by jgilliam17 — APPROVED** [review]
Thank you @hardikmodi1 
Tested, looks good to merge.
- Verified post creation - as expected.

**Pull request merged**